ports/7zip/portfile.cmake

```
@@ -28,3 +28,5 @@ file(
     DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"
     RENAME copyright
 )
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```

```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")

```

Please leave a blank line at the end of the file.